addons/account/models/ir_module.py

```diff
@@ -87,3 +87,14 @@ def _register_hook(self):
                 self.env.company,
             )
             del self.env.registry._auto_install_template
+
+    def module_uninstall(self):
```

Comment:
add a comment here about what we do and why

Reply:
done.